src/__test__/App.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import store from '../redux/configureStore';
+import App from '../App';
+
+describe('App component', () => {
+  test('renders', () => {
+    const tree = render(
```

Comment:
- [ ] Please Consider adding to your App more unit tests for several components of the app and some pure functions in Redux code. 👨🏻‍🎨